rt/src/runtime.rs

```diff
@@ -22,13 +22,17 @@ use crate::process::{NativeAsyncMethod, Process};
 use crate::scheduler::{number_of_cores, pin_thread_to_core};
 use crate::stack::Stack;
 use crate::state::{MethodCounts, RcState, State};
+
 use std::env::args_os;
 use std::io::{stdout, Write as _};
 use std::process::exit as rust_exit;
 use std::thread;
 
 #[cfg(unix)]
```

Comment:
```suggestion
```

This only applies to the `pub type` (or whatever statement comes directly after it), which in this case isn't useful (and not needed, see my below comment for that).